packages/paste-website/src/pages/page-templates/wizard/index.mdx

```diff
@@ -43,7 +47,189 @@ export const getStaticProps = async () => {
+  storyID="website-pagetemplateexamples--default-wizard-with-content-example"
+  code={DefaultWizardWithContentExample}
+/>
+
+### Content structure
+
+Before deciding on a page layout, make a few content and experience decisions first. This includes:
+
+- **Wizard title:** Describes the overall goal the customer is trying to achieve. The title should start with a verb. Examples: “Upgrade”, “Build Track call”.
+- **Task headings:** For each page, the task heading describes the current task the customer is doing. Each heading should generally be in the format “[Verb] [object]”. Examples: “Select computed trait”, “Connect to Whatsapp”.
+- **Task content:** Decide what content and components will help a customer finish each task.
+- **Goal linearity and duration:** Decide whether a customer can achieve the entire goal in one session. Decide whether they need to leave Twilio to finish a task and whether they need to wait for Twilio to do something before finishing the flow.
+
+There are 3 standard max-widths for the content of each Wizard page. Based on the content you need for each task, start with the smallest max-width and size up as needed:
+1. `size-70` (712px). If this is too small, use:
+2. `size-100` (1024px). If this is too small, use:
+3. Full-width, with a max-width of `size-70` for the Page Header. 
+
+Use the same max-width for each page of the Wizard if possible.
+
+### Default template
+
+In most scenarios, you’ll take customers through a Wizard without having to change the Sidebar or Topbar.
+
+<StoryPreview
+  title="Wizard page template - default with content"
+  storyID="website-pagetemplateexamples--default-wizard-with-content-example"
+  code={DefaultWizardWithContentExample}
+/>
+
+#### Page header
+
+At the top of each page of the Wizard, add:
+- Wizard title as [Detail Text](/components/detail-text) (optional)
+- Task [Heading](/components/heading)
+- [Paragraph](/components/paragraph) text under the heading to explain the step (optional)
+
+#### Body
+
+Use the [Form](/components/form) component to set spacing between form elements.
+
+There may be cases where you need to show multiple tasks on the same page because customers need to reference them at the same time. If this is the case, use a stylized ordered list (coming soon!) to group tasks.
+
+<StoryPreview
+  title="Wizard page template - default ordered list"
+  storyID="website-pagetemplateexamples--default-wizard-ordered-list-example"
+  code={DefaultWizardOrderedListExample}
+/>
+
+<Callout variant='neutral' marginY="space70">
+  <CalloutHeading as="h2">Looking for guidelines on progressive disclosures?</CalloutHeading>
+  <CalloutText>We don’t have guidelines yet for this part of Wizard. If you need it for your work, please consider partnering with us to contribute it.</CalloutText>
+</Callout>
+
+#### Actions footer
+
+Use the [Form](/components/form) component to set spacing between form actions and form elements.
```

Comment:
todo: remove this sentence.

whoopsie, we can't use Form to right-align actions lol